packages/kit/src/runtime/server/fetch.js

```diff
@@ -131,6 +131,7 @@ export function create_fetch({ event, options, manifest, state, get_cookie_heade
 					);
 				}
 
+				state.initiator = event.request.url;
```

Comment:
putting it here undoes this line, causing an infinite loop in the case where `event.fetch` happens in a root layout while rendering an error page https://github.com/sveltejs/kit/blob/80a15056d3e7d36ec35bb4339afe3e5c5d8d45f4/packages/kit/src/runtime/server/page/respond_with_error.js#L46

Reply:
Good catch, this needs a `if not generic error initiator ` check then